docs/UserGuide.md

```diff
@@ -126,8 +126,13 @@ Refer to the [Features](#3-features) section for details of each command.
 * `STUDENT_ID` should be in the following format: **"A" + (7 digits) + (1 uppercase letter)**. <br>
   e.g. `A0123456G`.
 
-* `FYP_NAME` and `DEADLINE_NAME` should only include alphanumeric characters and spaces but
+* `STUDENT_NAME` should only contain alphabetic characters, spaces and slash(for Indian names).
+  **cannot start with a space**.<br>
```

Comment:
Do check if this is a typo or a grammatical/punctuation error. UG is strict when it comes to this.